packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -329,7 +329,10 @@ describe( 'Default search suggestions', () => {
 		const searchInput = container.querySelector( 'input[aria-label="URL"]' );
 
 		// TODO: select these by aria relationship to autocomplete rather than arbitary selector.
-		const initialSearchResultElements = container.querySelectorAll( '[role="listbox"] [role="option"]' );
+		const searchResultsWrapper = container.querySelector( '[role="listbox"]' );
+		const initialSearchResultElements = searchResultsWrapper.querySelectorAll( '[role="option"]' );
+
+		const searchResultsLabel = container.querySelector( `#${ searchResultsWrapper.getAttribute( 'aria-labelledby' ) }` );
```

Comment:
If something did get changed with `aria-labelledby`, this may need to be changed as well.

Reply:
Tests pass because this attr is available when dealing with initial suggestions. So we should be all good!